cmd/minishift/cmd/start.go

```diff
@@ -414,6 +416,16 @@ func validateOpenshiftVersion() {
 func setSubscriptionManagerParameters() {
 	cluster.RegistrationParameters.Username = viper.GetString(startFlags.Username.Name)
 	cluster.RegistrationParameters.Password = viper.GetString(startFlags.Password.Name)
+	cluster.RegistrationParameters.GetUsernameInteractive = getUsernameInteractive
+	cluster.RegistrationParameters.GetPasswordInteractive = getPasswordInteractive
+}
+
+func getUsernameInteractive() string {
+	return inputUtils.ReadInputFromStdin("Username to register to subscription-manager")
+}
+
+func getPasswordInteractive() string {
+	return inputUtils.ReadPasswordFromStdin("Password to register to subscription-manager")
```

Comment:
Same as above. We want the registrar to specify the message. If we want we can add a check for empty string in which case a default this default message is used.